data/03-10-svm.php

```diff
@@ -45,13 +45,453 @@
+				sobre ou abaixo de H2 pertence à classe -1 (ambos são mostrados na Figura 14).
+			</p>
+			<p>
+				H1: <i>w<sub>1</sub>x<sub>1</sub> + w<sub>2</sub>x<sub>2</sub> + b</i> = 1
+			</p>
+			<p>
+				H2: <i>w<sub>1</sub>x<sub>1</sub> + w<sub>2</sub>x<sub>2</sub> + b</i> = -1
+			</p>
+			<p>
+				Observando a Figura 14, podemos notar que temos dois pontos, um quadradinho e uma bolinha, que são os 
+				mais próximos de H1 e H2, respectivamente. Eles são chamados de <strong>Support Vectors</strong> (<span class="small-caps">SVs</span>). 
+				O objetivo do <span class="small-caps">SVM</span> é exatamente encontrar esses <span class="small-caps">SV’s</span> e, por consequência, encontrar a <span class="small-caps">MMH</span>. 
+			</p>
+			<p>
+				Treinar o SVM, na verdade, envolve estimar os parâmetros w e b do modelo. Eles devem ser escolhidos de 
+				modo que as duas condições sejam satisfeitas:
+			</p>
+			<p>
+				wx<sub>i</sub> + b ≥ 1	    se y<sub>i</sub> = 1
+			</p>
+			<p>
+				wx<sub>i</sub> + b ≤ -1	    se y<sub>i</sub> = -1
+			</p>
+			<P>
+				Essas inequações acima podem ser reescritas como:
+			</P>
+			<p>
+				y<sub>i</sub> (wx<sub>i</sub> + b) ≥ 1
+			</p>
+			<p>
+				Não seria difícil resolvê-la, mas o <span class="small-caps">SVM</span> impõe um requisito adicional: a margem deve ser máxima! 
+				E maximizar a margem equivale a minimizar a seguinte função objetivo:
+			</p>
+			<p>
+				f(w) = ||w||<sup>2</sup> / 2
+			</p>
+			<p>
+				Então, temos o problema final definido:
+			</p>
+			<p>
+				Minimizar ||w||<sup>2</sup> / 2
+			</p>
+			<p>
+				s.a.  y<sub>i</sub> (wx<sub>i</sub> + b) ≥ 1
+			</p>
+			<p>
+				A função objetivo é quadrática e as restrições lineares em <i>w</i> e <i>b</i>. Trata-se de um problema de otimização 
+				convexo, resolvido pelo método padrão “multiplicador de Lagrange” 
+				(consulte <a href="https://see.stanford.edu/materials/aimlcs229/cs229-notes3.pdf"> 
+				https://see.stanford.edu/materials/aimlcs229/cs229-notes3.pdf</a> para obter detalhes). 
+				Não é algo trivial de ser resolvido, mas felizmente os softwares estatísticos costumam ter pacotes 
+				para resolver problemas deste tipo.
+			</p>
+		</section>
+
+		<section class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
+			<h2>Etapa de Classificação</h2>
+			<p>
+				O produto final do treinamento de um classificador <span class="small-caps">SVM</span> 
+				é uma equação que define a <span class="small-caps">MMH</span> e é baseada na formulação Lagrangiana:
+			</p>
+			
+			<!-- Não sei como passar para o html -->
+			<p>d(z)=∑_(i=1)^l▒〖yi×αi×xi×t+bo〗</p>
+			<p>
+				É a partir dessa fórmula que podemos classificar novos objetos. Basicamente, basta “plugar” as 
+				informações de um novo objeto (representado na fórmula por <i>t<i>). Se o resultado for positivo, o 
```

Comment:
Faltou fechar a tag i

```suggestion
				informações de um novo objeto (representado na fórmula por <i>t</i>). Se o resultado for positivo, o 
```